generators/app/templates/src/2.x/component/_component.html

```diff
@@ -1,21 +1,178 @@
 <% if (elementImplementation === 'bower') { -%>
 <link rel="import" href="../polymer/polymer-element.html">
 <% } else { -%>
-  <link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
```

Comment:
i updated the 2.0 component to only be the html file, which is how we are moving.